### PR TITLE
Update source paths in README

### PR DESCRIPTION
**Pull request by danra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,15 +74,15 @@ Cppfront builds with any major C++20 compiler.
 
 #### MSVC build instructions
 
-    cl cppfront.cpp -std:c++20 -EHsc
+    cl source\cppfront.cpp -std:c++20 -EHsc
```
Comment (gregmarr) [inline README.md:77]: backslash instead of forward slash

**Comment by danra** [comment]
It's probably preferrable for the binary to be built outside of the source directory. So just specifiying the path makes sense to me. It's also less "explaining" and more "just works" than the alternative of instructing to change into the source folder.

**Comment by gregmarr** [comment]
On the other hand, I'd hate to imply that you HAVE to build it from the root of the install if it can be built from inside the directory.

**Comment by danra** [comment]
> On the other hand, I'd hate to imply that you HAVE to build it from the root of the install if it can be built from inside the directory.

I don't think that's implied... these are instructions for first-time comers to the repo. Having it "just work" makes it just a bit easier, probably saving approx. a minute.

**Comment by hsutter** [comment]
Thanks!

**Pull request merged**